src/ServicePulse.Host/app/js/services/services.service-control.js

```diff
@@ -40,9 +40,13 @@
 
         var previousExceptionGroupEtag;
 
-        function getExceptionGroups(classifier, classifierFilter) {
+        function getExceptionGroupsPromise(classifier, classifierFilter) {
```

Comment:
We return promises everywhere else here, this shouldn't be the only method with `Promise` in it's name. The convention should be that everything in this class returns a promise.